### ISSUE TITLE
Cannot read property 'setOption'of undefined

### ISSUE THREAD

**Issue by pjdohertygis** [Esri/Survey123Community#562]
I tried launching a form today from Survey123 Web and was given an error message, "Oops! - Failed to publish the survey."

However, upon inspection with the Chrome console I see "Cannot read property 'setOption'of undefined" - see image below. 

![image](https://cloud.githubusercontent.com/assets/4256979/18819798/891a705a-83f3-11e6-9ba6-c2d6e53aaeb5.png)

**Notice** - I was trying to add some hyperlinks via HTML in the Description - would this cause an issue in publishing? It also appears it will not save. 

Here is how I was trying to add links, although it would be nice to have a more advanced editing interface like in other ArcGIS Online apps. 

Below are some free resources already being used  Globally and here in New Zealand to support Emergency Management. 

<a href="http://solutions.arcgis.com/gallery/#s=0&md=industries:emergency-management" target="_blank">ArcGIS Solutions for Emergency Management</a>
These examples provide configurations oriented around the four phases of emergency management to facilitate work in preparedness, response, recovery, and mitigation. You can try out the applications, watch a video, and get step-by-step guides for implementing these apps. 

<a href="http://www.arcgis.com/home/group.html?id=394a043fdd2d4344bec2c3a79410efec" target="_blank">ArcGIS Service Catalog for Emergency Management</a> 
A collection of services used to publish layers in your ArcGIS Online organizational account and jumpstart your ArcGIS for Emergency Management implementation.

<a href="http://solutions.arcgis.com/community/#Emergency Management/" target="_blank">ArcGIS Community for Emergency Management</a> 
Connect with your peers and the teams at Esri that are focused on GIS in Public Safety.

**Comment by tedrick** [comment]
Hi there,

I'm able to publish with HTML < a > tags in the description, so I don't think that's the particular issue.  Quick question- do you have a character for your description, footer & submission message?  Also, how many questions are on the survey?

**Comment by zhif7782** [comment]
Hi @pjdohertygis 
Thanks for touching us in email and provide the detail html code you're using.

```
<a href="http://solutions.arcgis.com/gallery/#s=0&md=industries:somename" target="_blank">ArcGIS Solutions for Emergency Management</a>
These examples provide configurations oriented around the four phases of emergency management to facilitate work in preparedness, response, recovery, and mitigation. You can try out the applications, watch a video, and get step-by-step guides for implementing these apps. 

<a href="http://www.arcgis.com/home/group.html?id=somegroupid" target="_blank">ArcGIS Service Catalog for Emergency Management</a> 
A collection of services used to publish layers in your ArcGIS Online organizational account and jumpstart your ArcGIS for Emergency Management implementation.

<a href="http://solutions.arcgis.com/community/#Emergency Management/" target="_blank">ArcGIS Community for Emergency Management</a> 
```

I can reproduce the issue by using the code above. When I click "Save" button in Design page, I notice the server side has returned some error message in addition to the error output in console:
![image](https://cloud.githubusercontent.com/assets/7665168/18842185/d3e841ea-8446-11e6-8fdb-96d43c85ba7b.png)

which exposes the exact reason of the issue: there're some unsupported characters by the server in the html code.

This does not mean that you cannot use the links, but it's a bug we have not sanitized the html code before saving.

We've logged this bug in our dev repo, and hopefully can fix it soon, will let you know as soon as it's fixed.

BTW, thanks to @tedrick , it's exact the reason you point out.

**Comment by zhif7782** [comment]
Hi @pjdohertygis , 
After some tries, I found it's the `=` (equals) prevent from saving the survey successfully. If I just remove the `=` in your html code, the survey could be saved successfully.

**Comment by survey123** [comment]
Rich text content editing has been introduced in the web designer and will be available in 2.3 release, it's available in following sections:
* Survey header
* Survey description
* Question hint
* Note question
